client/src/components/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import { ButtonHTMLAttributes, ReactNode } from "react";
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children: ReactNode;
+  addNewContent?: string;
+}
+
+/**
+ *
+ * @param children The button content.
+ * @param addNewContent A function to add new styles to the button. If you use `className`, it will overwrite all the styles.
+ * @returns A button with custom styles or default styles.
+ */
+
+export default function Button({
```

Comment:
this was necessary?

Reply:
Yes, to have a custom button so we don't need to write the code over and over again for every button on the page, and the comments above are to make it clear about the button.